src/popups/HandicapPopup/HandicapPopup.js

```diff
@@ -52,8 +53,10 @@ function HandicapPopup({ feature }) {
 
   const equipmentStr = equipment.length ? (
     <div className="wkp-handicap-popup-element" key="Ausstattung">
-      <div className="wkp-handicap-popup-field-title">{t('Ausstattung')}</div>
-      <div className="wkp-handicap-popup-field-body">
+      <div className="wkp-handicap-popup-field-title" tabIndex={0}>
+        {t('Ausstattung')}
+      </div>
+      <div className="wkp-handicap-popup-field-body" tabIndex={0}>
```

Comment:
What about the aria role attribute for heading, link, etc.?

Reply:
aria role heading added to the titles